content/en/docs/installation/installation-linux.md

```diff
@@ -20,30 +20,30 @@ Please modify the commands accordingly.
 
 {{< tabs >}}
 {{% tab name="Ubuntu" %}}
+
 ```bash
 git clone https://github.com/green-coding-berlin/green-metrics-tool ~/green-metrics-tool && \
 cd ~/green-metrics-tool && \
-git submodule update --init && \
 sudo apt update && \
 sudo apt upgrade -y && \
 sudo apt install make gcc python3 python3-pip libpq-dev -y && \
```

Comment:
You could add `libglib2.0-dev` here so the error described above doesn't happen?